includes/library/zencart/Services/src/LeadGroupPricingRoutes.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Class LeadGroupPricingRoutes
+ *
+ * @copyright Copyright 2003-2016 Zen Cart Development Team
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version GIT: $Id: Author: Ian Wilson  Fri Aug 17 17:42:37 2012 +0100 New in v1.5.1 $
+ */
+namespace ZenCart\Services;
+
+/**
+ * Class LeadGroupPricingRoutes
+ * @package ZenCart\Services
+ */
+class LeadGroupPricingRoutes extends LeadRoutes
+{
+    /**
+     * @return array
+     */
+    public function deleteCheck()
+    {
+        $retVal = array(true, '');
+        $sql = "SELECT count(*) as total FROM " . TABLE_CUSTOMERS . " WHERE customers_group_pricing = :id:";
```

Comment:
Currently no. 
The route/service stuff was meant to get around the problem of FAT controllers, and as such they aren't available to the listingQueryAndOutput classes. So there is no current simple way to share code (apart from writing  and using functions)

There is a solution to the general problem of reuse of queries, and that is to have a model/repository class/factory that is shared (instead of sharing the $db ) either directly or via an internal api.